docs/gh_pages/docs/config.md

```diff
@@ -36,13 +36,14 @@ daemon:
   port: 8000
   host: localhost
 logging:
+
   level: info
 reports:
   format: pdf
   renderer: xhtml2pdf
   outputDir: ~/.pebblo
 classifier:
-  anonymizeAllEntities: true
+  anonymizeSnippets: false
```

Comment:
`false` works or do we need to put it as `False`? Developers will copy this and customize it based on their needs.